### ISSUE TITLE
Operator docs may require enhancement

### ISSUE THREAD

**Issue by Vacant2333** [karmada-io/karmada#3963]
**What would you like to be added**:
Operator docs may require enhancement
https://github.com/karmada-io/karmada/blob/master/operator/README.md

**Why is this needed**:
for example:
we can add the the usage to get the karmada-apiserver's kubeconfig
`kubectl get secret -n {namespace} karmada-demo-admin-config -o jsonpath='{.data.config}' | base64 --decode > kubeconfig.yaml`

**Comment by RainbowMango** [comment]
Sounds good to me. @calvin0327 can you help to confirm that?

**Comment by calvin0327** [comment]
@RainbowMango That's sure.

/assign

**Comment by calvin0327** [comment]
@Vacant2333  Thanks pointing this, there is a better way to get the karmada config:

```shell
kubectl  get secret -n <KARMADA_NAMESPACE> $(kubectl get karmada -n <KARMADA_NAMESPACE> <KARMADA_NAME> -ojsonpath='{.status.secretRef.name}') -ojsonpath='{.data.config}' | base64 -d > config
```

**Comment by Vacant2333** [comment]
> @Vacant2333  Thanks pointing this, there is a better way to get the karmada config:
> 
> 
> 
> ```shell
> 
> kubectl get secret -n <KARMADA_NAMESPACE> $(kubectl get karmada -n <KARMADA_NAMESPACE> <KARMADA_NAME> -ojsonpath='{.status.secretRef.name}') -ojsonpath='{.data.config}' | base64 -d > config
> 
> ```
> 
> 

yes its better，and is there something we can add to the file too？

**Comment by RainbowMango** [comment]
@calvin0327, @Vacant2333 might want to help on this :)

**Comment by calvin0327** [comment]
> > @calvin0327, @Vacant2333 might want to help on this :)
> 
> ah no, I mean if there something can update to this file too that I didnt mention. And it's ok Calvin to do this

Except for this, I didn't think of anything else to update to the README.

**Comment by Vacant2333** [comment]
> > > @calvin0327, @Vacant2333 might want to help on this :)
> > 
> > 
> > ah no, I mean if there something can update to this file too that I didnt mention. And it's ok Calvin to do this
> 
> Except for this, I didn't think of anything else to update to the README.

we may can add a example for `spec.hostCluster.secretRef` and some links to the docs(component), its a suggestion, im not sure if we need
